application/testing/CMakeLists.txt

```diff
@@ -662,13 +703,13 @@ f3d_test(NAME TestInvalidFullScene DATA invalid.gltf ARGS --verbose REGEXP "Load
 f3d_test(NAME TestInvalidTexture DATA cow.vtp ARGS --texture-material=${F3D_SOURCE_DIR}/testing/data/invalid.png REGEXP "Cannot open texture file" NO_BASELINE)
 
 # Test non existent HDRI, do not add a dummy.png
-f3d_test(NAME TestNonExistentHDRI DATA cow.vtp ARGS --hdri=${F3D_SOURCE_DIR}/testing/data/dummy.png REGEXP "HDRI file does not exist" NO_BASELINE)
+f3d_test(NAME TestNonExistentHDRI DATA cow.vtp ARGS --hdri-file=${F3D_SOURCE_DIR}/testing/data/dummy.png REGEXP "HDRI file does not exist" NO_BASELINE)
```

Comment:
that would disable the test on many CI, I dont think we want that.

Reply:
Maybe we should reduce the size of HDRIs in the testing files to make sure the tests run faster and complete quicker on macOS runners. Then we can get rid of the flag to disable HDRI tests.

Reply:
TestHDRI without caches is taking 30s on linux. We would need to use a really really small HDRI to be able to run these tests on MacOS, I'm afraid it would reduce the quality of testing.